package.json

```diff
@@ -53,8 +53,8 @@
     "mocha": "^5.0.4",
     "mocha-chrome": "^1.0.3",
     "moment": "~2.21.0",
-    "ms-rest-azure-js": "^0.3.0",
-    "ms-rest-js": "^0.4.0",
+    "ms-rest-azure-js": "github:Azure/ms-rest-azure-js",
+    "ms-rest-js": "github:Azure/ms-rest-js",
```

Comment:
Yeah. Amar wants me to hold off on releasing ms-rest-js 0.5.0 until we've had a couple of weeks with the storage team's feedback, so that means that I have to do GitHub references until then.